pyproject.toml

```diff
@@ -13,6 +13,7 @@ dependencies = [
     "aiozoneinfo>=0.1",
     "arrow>=1.2",
     "pyotp>=2.0",
+    "cryptography>=43.0.0",
```

Comment:
Move it one line above so that this list is ordered.

Reply:
Done!